src/main/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+### direct log messages to stdout ###
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
```

Comment:
We cannot accept adapter with logging to the std.out. Such logs will go to Tomcat's catalina.out and that file with overgrow - its size is not managed. Adapter should use logging configuration from Symphony which will log to our log files.